test-collateral/Makefile

```diff
@@ -103,8 +103,11 @@ DATA_FILES = linear-regression.dat \
 			 number-stream-1.dat \
 			 number-stream-2.dat
 
+DATA_DIRS = idash2017 \
+			macd \
```

Comment:
Nit: some bizarre indentation here.  Can you fix?

Reply:
It was the space vs tab. The tab is interpreted in different lengths. I changed all to space.